src/config.js

```diff
@@ -26,6 +26,7 @@ module.exports = (env) => {
       logLevel: 'silent',
       https: env.https || false,
       overlay: true,
+      host: '0.0.0.0',
```

Comment:
This configuration could be passed as an argument instead.

```suggestion
      host: env.host || 'localhost',
```